### PR TITLE
issue #376

### PR DESCRIPTION
**Pull request by paras-jain**
fixed some links that were not working.

### CODE REVIEW

File: docs/emoji-key.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ sidebar_label: Emoji Key ✨
 
 ## Table
 
-> To have a contribution added when using the [Bot](bot/overview) or [CLI](cli/overview) use the keyword in the `Type` column. The bot will also use [basic Natural Language Parsing](https://github.com/all-contributors/all-contributors-bot/blob/master/src/tasks/processIssueComment/utils/parse-comment/index.js) to determine your contribution intent.
+> To have a contribution added when using the [Bot](https://github.com/paras-jain/all-contributors/blob/master/docs/bot/overview.md) or [CLI](https://github.com/paras-jain/all-contributors/blob/master/docs/cli/overview.md) use the keyword in the `Type` column. The bot will also use [basic Natural Language Parsing](https://github.com/all-contributors/all-contributors-bot/blob/master/src/tasks/processIssueComment/utils/parse-comment/index.js) to determine your contribution intent.
```
Comment (Berkmann18) [inline docs/emoji-key.md]: Please use refrain from linking to your repo instead of the official one.

File: docs/emoji-key.md [hunk 2]
```diff
@@ -41,5 +41,5 @@ Emoji/Type | Represents | Comments
 📹 <br /> `video` | Videos | links to the video
 
 ## What's next
-- [Notes for repository maintainers](repository-maintainers)
-- [Automating the process using tools](tooling)
+- [Notes for repository maintainers](https://github.com/all-contributors/all-contributors/blob/master/docs/repository-maintainers.md)
+- [Automating the process using tools](https://github.com/all-contributors/all-contributors/blob/master/docs/tooling.md)
```
Comment (Berkmann18) [inline docs/emoji-key.md]: Absolute paths are unnecessary here and will inevitably break the Docusaurus build.

File: docs/overview.md [hunk 3]
```diff
@@ -7,7 +7,7 @@ sidebar_label: Overview
 > Call for translators! [We're looking for translators](https://github.com/all-contributors/all-contributors/issues/143) to help translate this spec for everyone!
 
 <div align="center">
-    <img src="../assets/logo-full-transparent.png" alt="✨ All Contributors ✨" width="800px" />
+    <img src="../docs/assets/logo-full-transparent.png" alt="✨ All Contributors ✨" width="800px" />
```
Comment (Berkmann18) [inline docs/overview.md]: Have you checked if the deployed version still shows the image like on https://allcontributors.org/docs/en/overview?

File: docs/overview.md [hunk 4]
```diff
@@ -23,14 +23,14 @@ People are giving themselves and their free time to contribute to open source pr
 ## The All Contributors Table
 Below is an example of how using the all-contributors spec table can recognize all contributors
 <div align="center">
-    <img src="../assets/contributors-table-small.png" alt="All Contributors Table Screenshot" width="800px" />
+    <img src="https://raw.githubusercontent.com/all-contributors/all-contributors/master/docs/assets/contributors-table-small.png" alt="All Contributors Table Screenshot" width="800px" />
```
Comment (Berkmann18) [inline docs/overview.md]: Same comment as above.

File: docs/overview.md [hunk 5]
```diff
@@ -23,14 +23,14 @@ People are giving themselves and their free time to contribute to open source pr
...
 
 ## What's next
-- [The all contributors specification](specification)
-- [Emoji Key (Contribution Types Reference)](emoji-key)
-- [Notes for repository maintainers](repository-maintainers)
-- [Automating the process using tools](tooling)
+- [The all contributors specification](https://github.com/paras-jain/all-contributors/blob/master/docs/specification.md)
+- [Emoji Key (Contribution Types Reference)](https://github.com/paras-jain/all-contributors/blob/master/docs/emoji-key.md)
+- [Notes for repository maintainers](https://github.com/paras-jain/all-contributors/blob/master/docs/repository-maintainers.md)
+- [Automating the process using tools](https://github.com/paras-jain/all-contributors/blob/master/docs/tooling.md)
```
Comment (Berkmann18) [inline docs/overview.md]: Please use refrain from linking to your repo instead of the official one.

File: docs/repository-maintainers.md [hunk 6]
```diff
@@ -7,7 +7,7 @@ sidebar_label: Repository Maintainers
 Contributors should be recognised as soon as the contribution is discovered to help ensure their efforts are not
 overlooked when the list gets updated at a later date. This can be difficult for contributions that are not the result of a commit to the repository but do your best to minimise the time between the contribution and updating the list.
 
-> **Pro tip:** If your project is configured for [the @all-contributors bot 🤖](bot/overview) simply write a comment on an issue or pull request to recognise their contribution. For example:
+> **Pro tip:** If your project is configured for [the @all-contributors bot 🤖](https://github.com/paras-jain/all-contributors/blob/master/docs/bot/overview.md) simply write a comment on an issue or pull request to recognise their contribution. For example:
```
Comment (Berkmann18) [inline docs/repository-maintainers.md]: Same comment as above.

File: docs/repository-maintainers.md [hunk 7]
```diff
@@ -25,4 +25,4 @@ In the end, there are no hard and fast rules for when a contribution has to be a
 just do your best to be fair and to ensure all contributors are recognised.
 
 ## What's next
-- [Automating the process using tools](tooling)
+- [Automating the process using tools](https://github.com/paras-jain/all-contributors/blob/master/docs/tooling.md)
```
Comment (Berkmann18) [inline docs/repository-maintainers.md]: Same comment as above.

File: docs/specification.md [hunk 8]
```diff
@@ -6,7 +6,7 @@ sidebar_label: Specification
 
 An example of how using the all-contributors spec table can recognize all contributors
 <div align="center">
-    <img src="../assets/contributors-table-small.png" alt="Contributors Table Screenshot" width="800px" />
+    <img src="https://raw.githubusercontent.com/all-contributors/all-contributors/master/docs/assets/contributors-table-small.png" alt="Contributors Table Screenshot" width="800px" />
```
Comment (Berkmann18) [inline docs/specification.md]: Is it necessary to use the RAW GH URL for this?

File: docs/specification.md [hunk 9]
```diff
@@ -19,12 +19,12 @@ Open source projects should include the following mandatory items in order to su
     - Name
     - URL link to a site where more information can be learned about the contributor.  This URL may be determined by the contributor at the project's discretion.
     - An indication of the Contribution Category in text or icon image format using the defined Contribution Categories and/or Contribution Categories emoji images (see below).
-    - A link to the Contribution Category in text or emoji format using the defined Contribution Categories and/or [Contribution Categories emoji](emoji-key).
+    - A link to the Contribution Category in text or emoji format using the defined Contribution Categories and/or [Contribution Categories emoji](https://github.com/paras-jain/all-contributors/blob/master/docs/emoji-key.md).
```
Comment (Berkmann18) [inline docs/specification.md]: Please use refrain from linking to your repo instead of the official one.

File: docs/specification.md [hunk 10]
```diff
@@ -19,12 +19,12 @@ Open source projects should include the following mandatory items in order to su
...
 
-> **Pro tip:** You can use [the @all-contributors bot 🤖](bot/overview) to automate acknowledging contributors to your open source projects ✨
+> **Pro tip:** You can use [the @all-contributors bot 🤖](https://github.com/paras-jain/all-contributors/blob/master/docs/bot/overview.md) to automate acknowledging contributors to your open source projects ✨
```
Comment (Berkmann18) [inline docs/specification.md]: Please use refrain from linking to your repo instead of the official one.

File: docs/specification.md [hunk 11]
```diff
@@ -33,7 +33,7 @@ Projects should consider the following optional items as part of their commitmen
 3. An optional link from the Contribution Category indicator to the category-specific contribution(s) can be included to provide additional information about the project contribution(s) as detailed in the section below.
 
 ## What's next
-- [Emoji Key (Contribution Types Reference)](emoji-key)
-- [Notes for repository maintainers](repository-maintainers)
-- [Automating the process using tools](tooling)
+- [Emoji Key (Contribution Types Reference)](https://github.com/paras-jain/all-contributors/blob/master/docs/emoji-key.md)
+- [Notes for repository maintainers](https://github.com/paras-jain/all-contributors/blob/master/docs/repository-maintainers.md)
+- [Automating the process using tools](https://github.com/paras-jain/all-contributors/blob/master/docs/tooling.md)
```
Comment (Berkmann18) [inline docs/specification.md]: Same comment as above.

File: docs/tooling.md [hunk 12]
```diff
@@ -5,14 +5,14 @@ sidebar_label: Tooling
...
+Enter the [@all-contributors bot 🤖](https://github.com/paras-jain/all-contributors/blob/master/docs/bot/overview.md)! The bot will automatically pull a user's profile, grab the contribution type emoji, update the project README and then open a Pull Request against the project ✨
 <a href="bot/usage">
-    <img alt="Example usage screenshot" src="../assets/bot-usage.png" width="500px">
+    <img alt="Example usage screenshot" src="https://github.com/all-contributors/all-contributors/blob/master/docs/assets/bot-usage.png" width="500px">
 </a>
 
 
 ### All Contributors CLI
-If you prefer command line tools there is also the [all-contributors CLI tool](cli/overview) to help automate adding contributor acknowledgements.
+If you prefer command line tools there is also the [all-contributors CLI tool](https://github.com/paras-jain/all-contributors/blob/master/docs/cli/overview.md) to help automate adding contributor acknowledgements.
```
Comment (Berkmann18) [inline docs/tooling.md]: Same comment as above.

File: docs/emoji-key.md (hunk 1 above)
Comment (paras-jain) [reply docs/emoji-key.md]: will keep that in mind.

File: docs/emoji-key.md (hunk 2 above)
Comment (paras-jain) [reply docs/emoji-key.md]: I had completely forgotten about the docusaurus build, sorry. The links in the md were not working; is there no way we can make the links work without appending .md manually?

File: docs/overview.md (hunk 3 above)
Comment (paras-jain) [reply docs/overview.md]: The deployed version showed the image, but it was not visible in the markdown file.

File: docs/specification.md (hunk 8 above)
Comment (paras-jain) [reply docs/specification.md]: using raw gh url will make it visible on both the markdown and deployed version.

File: docs/emoji-key.md (hunk 2 above)
Comment (Berkmann18) [reply docs/emoji-key.md]: I don't think so.
@jakebolam @gr2m @sinchang Any ideas?

File: docs/overview.md (hunk 3 above)
Comment (Berkmann18) [reply docs/overview.md]: I would have suggested seeing what the Netlify preview shows but it's blocked due to the semantic PR check.

File: docs/specification.md (hunk 8 above)
Comment (Berkmann18) [reply docs/specification.md]: Good point.

**Pull request closed without merging**